Cargo.toml

```diff
@@ -11,3 +11,6 @@ categories = ["no-std"]
 
 [build-dependencies]
 rustc_version = "0.2.3"
+
+[dependencies]
+doc-comment = "0.3"
```

Comment:
@GuillaumeGomez is there any way to avoid making this a full dependency? I read the docs of doc-comment but found no mention of this.

Reply:
If used for tests, it's not supposed to be a full dependency. For example: https://github.com/rust-lang/hashbrown/pull/105/files

Reply:
Ah, but that needs the nightly-only `cfg(doctest)`.

Reply:
Yes. `cfg(test)` for rustdoc has been removed just before it hit nightly (it was in beta) and has been replaced with `cfg(doctest)`: https://github.com/rust-lang/rust/pull/63201 (don't hesitate to leave a comment there!)